src/main/com/fulcrologic/rad/database_adapters/sql_options.cljc

```diff
@@ -95,4 +95,8 @@
    isComponent flag.  This is different than SQL CASCADE, because in this case nothing
    has been deleted. The reference on which this option lives is just being changed
    to point to a different thing."
-  :com.fulcrologic.rad.database-adapters.sql/delete-referent?)
\ No newline at end of file
+  :com.fulcrologic.rad.database-adapters.sql/delete-referent?)
+
+(def data-type
+  "Attribute option. Force a data type for the desired attribute when auto generating the database."
```

Comment:
Change this to be a string, which is a database-specific data type that will be taken literally as an override.